apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -119,6 +119,22 @@ window.PLOrderBlocks = function (uuid, options) {
     }
   }
 
+  function drawIndentLocationLines(dropzoneElementId) {
+    // draw the grid lines maxIndent number of times
+    $(dropzoneElementId)[0].style.background = 'linear-gradient(#000, #000) no-repeat border-box, '
+      .repeat(maxIndent + 1)
+      .slice(0, -2);
+    // set the appearance (thickness) of the grid lines
+    $(dropzoneElementId)[0].style.backgroundSize = '1px 100%, '.repeat(maxIndent + 1).slice(0, -2);
+    // translate the grid lines to the correct positions
+    $(dropzoneElementId)[0].style.backgroundPosition = Array.from(
+      { length: maxIndent + 1 },
+      (value, index) => {
+        return `${+$(dropzoneElementId).css('padding-left').slice(0, -2) + TABWIDTH * index}px 0`;
+      }
+    ).join(', ');
+  }
```

Comment:
Looks much improved from before!

Minor improvements I would suggest: 
- I still don't think its necessary to use `linear-gradient`, can't you just make the background be a solid color? It should be the same things as giving it a constant gradient like here, but will make shorter, simpler code
- I don't really like these kinds of comments that just say exactly what the next line does, I think they just get in the way. Its easier to just read the code
- Why not just pass `$(dropzoneElementId)[0]` as the parameter to the function? That will clean up the code a little more

Reply:
I removed the comments as suggested. For the `linear-gradient` issue, I had a quick look, and it looks like we are stuck with either `linear-gradient` or using SVGs to draw the line. Solid background colours are simpler, but after a quick look it looks like we'll have to add more divs acting as lines to use that approach. I can't seem to find a function that is simpler than `linear-gradient` either.